tests/unit_tests/test_unit.py

```
@@ -186,6 +188,27 @@ def __call__(self, url):
+    mock_get = MockResponseGet()
+    mock_github = functools.partial(MockGithub, test_releases)
+
+    with monkeypatch.context() as mnk, tempfile.TemporaryDirectory() as tmp_dir:
+        mnk.setattr(casanovo, "__version__", "3.0.0")
+        mnk.setattr(github, "Github", mock_github)
+        mnk.setattr(requests, "get", mock_get)
+
+        filename = pathlib.Path(tmp_dir) / "casanovo_timstof.ckpt"
+        assert not filename.is_file()
+        _, result_path = casanovo.setup_model(
+            "timstof", None, None, None, False
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)